src/patternfly/components/Progress/progress.scss

```diff
@@ -18,7 +18,7 @@
   --pf-c-progress__bar--Height--sm: var(--pf-global--spacer--sm);
   --pf-c-progress__bar--Height--lg: var(--pf-global--spacer--lg);
   --pf-c-progress__bar-overlay--Opacity: .2; // one-off change in opacity to allow progress bar to automatically coordinate with the indicator color
-  --pf-c-progress__indicator--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-progress__indicator--MinWidth: 0;
```

Comment:
you can just ditch this variable if you don't want to pull it in on line 141. I don't think we need to set them for 0.

Reply:
Ah that makes sense. Updating now.